### DISCUSSION TITLE
Option to pin entries on top?

### DISCUSSION THREAD

**Discussion by patrickunterwegs** [TechbeeAT/jtxBoard#181] [Polls]
There was a feature request to be able to pin entries so that they always stay on top. 
The problem is that the standard does not have such a flag. But categories could be used - for example a category "✰" could be defined when an entry is is marked as pinned. But still this would add a new category to the entry which might not be desired. At the same time users can anyway assign a category and filter by it. Additionally other systems or apps will not know the "pinned" category and will not pin it on top. 

So the question is if it makes any sense to implement such a feature. What do you think?

**Comment by rogercreagh** [comment]
The only way it could make sense would be if it was implemented as a jtxB specific instance option.
ie for jtxB on a particular device allow one item (journal, note, task) to be pinned to the top of the each list but don't bother to sync it.

You could take this a step further an add an ordering field to items, which wasn't sync'd but just held locally and updated on sync if items were removed or added remotely (remove deleted ones from the order list, and add new ones at the bottom) but there would have to be a good use case demonstrated. (just because something is possible doesn't mean its a good idea - the classic trap of function creep)

It _might_ be useful to define an arbitrary order for Notes in particular - but you can achieve this now by prefixing a number to the title (summary) and sorting by that.
